src/GraphQL.Federation/GraphQL.Federation.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp3.0</TargetFrameworks>
+    <Description>Apollo Federation for GraphQL.NET</Description>
+    <PackageTags>GraphQL;Federation</PackageTags>
+    <OutputType>Library</OutputType>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\GraphQL\GraphQL.csproj" />
+    <PackageReference Include="Google.Protobuf" Version="3.14.0" />
```

Comment:
Federation is somehow related to Google.Protobuf?

Reply:
The Apollo federated tracing requires the Protobuf format, which is perhaps a bit unfortunate, but that is what they went with.

Reply:
https://www.apollographql.com/docs/federation/metrics/#how-tracing-data-is-exposed-from-a-federated-service

Reply:
OK.